node_modules/col-activities/lib/default.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2015 UC Berkeley (UCB) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var Default = module.exports = [
+  {
+    'type': 'add_asset',
+    'points': 5,
+    'enabled': true
+  },
+  {
+    'type': 'like',
+    'points': 1,
+    'enabled': true
+  },
+  {
+    'type': 'dislike',
+    'points': -1,
+    'enabled': true
+  },
+  {
+    'type': 'get_like',
```

Comment:
What does a `get_like` or `get_dislike` activity reflect?  "Getting the number of likes / dislikes for an asset?"